### ISSUE TITLE
Error ejecting fresh project with ExpoKit

### ISSUE THREAD

**Issue by amosyuen** [react-community/create-react-native-app#474]
## Description

I created a fresh project using create-react-native-app v1.0.0. Then I ran eject with ExpoKit.

## Expected Behavior

Expected eject to succeed.

## Observed Behavior

Got this error message:
```
Error running eject: Error: ENOENT: no such file or directory, open 'C:\Programming\react-native-app\io
s\react-native-app\Supporting\Info.plist'
```

## Environment

Please run these commands in the project folder and fill in their results:

* `npm ls react-native-scripts`: 1.7.0
* `npm ls react-native`: 0.49.5
* `npm ls expo`: 22.0.3
* `node -v`: 8.1.2
* `npm -v`: 5.0.3
* `yarn --version`: 1.2.1
* `watchman version`: N/A

Also specify:

1. Operating system: Windows 10
2. Phone/emulator/simulator & version:  N/A

## Reproducible Demo

Please provide a minimized reproducible demonstration of the problem you're reporting.

Issues that come with minimal repro's are resolved much more quickly than issues where a maintainer has to reproduce themselves.

**Comment by rexdogi** [comment]
experienced same error

**Comment by lihop** [comment]
I'm on Linux and encountered the same issue.
```bash
Validating project manifest...
It appears you are trying to detach outside of macOS. You will require Xcode to build the resulting artefact.
Downloading iOS code...
Moving iOS project files...
Naming iOS project...
Configuring iOS project...
Error running eject: Error: ENOENT: no such file or directory, open '/home/lihop/react-native-expo-starter-kit/ios/react-native-expo-starter-kit/Supporting/Info.plist'
```
I think it has something to do with the upgrade from xdl 45.0.0 to 46.0.1 in #446. The [XDL changelog](https://github.com/expo/xdl/blob/master/CHANGELOG.md#4600---2017-10-18) for 46.0.0 mentions this
> Remove dependency on macOS to detach
My workaround was to downgrade xdl to version 45.0.0 by running
```bash
yarn add xdl@45.0.0
```
and modifying yarn.lock like so
```diff
diff --git a/yarn.lock b/yarn.lock
index 07af53f..2624733 100644
--- a/yarn.lock
+++ b/yarn.lock
@@ -5519,7 +5519,7 @@ react-native-scripts@1.7.0:
     progress "^2.0.0"
     qrcode-terminal "^0.11.0"
     rimraf "^2.6.1"
-    xdl "46.0.1"
+    xdl "45.0.0"
 
 react-native-snap-carousel@2.4.0:
   version "2.4.0"
```
Then when I run `npm run eject` and follow the prompts to eject with ExpoKit I get the message
```bash
Validating project manifest...
Can't create an iOS project since you are not on macOS. You can rerun this command on macOS in the future to add an iOS project. Enter 'yes' to continue and just create an Android project:
```
Then I enter `yes` and the script successfully ejects the app for Android only.

**Comment by terribleben** [comment]
Hi, the current version of XDL [doesn't try to generate an iOS project outside macOS](https://github.com/expo/universe/blob/master/dev/xdl/src/detach/Detach.js#L146-L155). Can you try updating and see if it still happens?

**Comment by adwigren** [comment]
@terribleben I ran into the same error and solved it by updating the xdl version to 46.1.0. [xdl changelog](https://github.com/expo/xdl/blob/master/CHANGELOG.md#4610---2017-10-27)

### Changed

- Only detach iOS on macOS or when a --force flag is added
